### ISSUE TITLE
How to train or Fine-tuning a specific class on the AVA-Kinetics

### ISSUE THREAD

**Issue by Deep-learning999** [open-mmlab/mmaction2#853]
Before raising a question, you may need to check the following listed items.

**Checklist**

1. I have searched related issues but cannot get the expected help.
2. I have read the [FAQ documentation](https://mmaction2.readthedocs.io/en/latest/faq.html) but cannot get the expected help.

I want to use slowfast's pre-training on Kinetics700 or ava to fine-tune specific categories on ava_kinetics, how to organize the data set, how to modify the data import and configuration files

**Comment by Deep-learning999** [comment]
@kennymckormick @irvingzhang0512

**Comment by irvingzhang0512** [comment]
I'm not sure whether the annotations for AVA-Kinetics could be directly used for MMAction2.
If the annotations are ready to use, you can refer to this [config](https://github.com/open-mmlab/mmaction2/blob/master/configs/detection/ava/slowonly_kinetics_pretrained_r50_4x16x1_20e_ava_rgb_custom_classes.py) and this [tutorial](https://github.com/open-mmlab/mmaction2/tree/master/configs/detection/ava#train-custom-classes-from-ava-dataset) to train specific classes

**Comment by Deep-learning999** [comment]
https://research.google.com/ava/download.html#ava_actions_download For specific differences, please see ava_kinetics_v1_0.tar.gz

**Comment by Deep-learning999** [comment]
> I'm not sure whether the annotations for AVA-Kinetics could be directly used for MMAction2.
> If the annotations are ready to use, you can refer to this [config](https://github.com/open-mmlab/mmaction2/blob/master/configs/detection/ava/slowonly_kinetics_pretrained_r50_4x16x1_20e_ava_rgb_custom_classes.py) and this [tutorial](https://github.com/open-mmlab/mmaction2/tree/master/configs/detection/ava#train-custom-classes-from-ava-dataset) to train specific classes

https://research.google.com/ava/download.html#ava_actions_download For specific differences, please see ava_kinetics_v1_0.tar.gz

**Comment by Deep-learning999** [comment]
@kennymckormick

**Comment by Deep-learning999** [comment]
@innerlee

**Comment by kennymckormick** [comment]
Currently, we do not support AVA-Kinetics, since our AVADataset can not load Kinetics annotations properly. We plan to support it in May. If you want to implement it urself, you may need to write a new dataset.

**Comment by Deep-learning999** [comment]
@irvingzhang0512

**Comment by Deep-learning999** [comment]
The Kinetics part of AVA-Kinetics has the staff box and action label in the key frame. The ava part is the same as ava2.1. How to convert the Kinetics part to the same format as ava2.1

**Comment by Deep-learning999** [comment]
@irvingzhang0512

**Comment by Deep-learning999** [comment]
> Currently, we do not support AVA-Kinetics, since our AVADataset can not load Kinetics annotations properly. We plan to support it in May. If you want to implement it urself, you may need to write a new dataset.
If I only download 3 categories of the ava data set for training or migration learning, are the custom configuration and tutorials you just posted apply?

**Comment by Deep-learning999** [comment]
If I only download 3 categories of the ava data set for training or migration learning, are the custom configuration and tutorials you just posted apply?
@irvingzhang0512 @innerlee @kennymckormick

**Comment by coldmanck** [comment]
@kennymckormick any update on supporting AVA-Kinetics? Thanks!

**Comment by coldmanck** [comment]
@kennymckormick appreciate if you'd like to provide any update on supporting AVA-Kinetics. Thanks!

**Comment by kennymckormick** [comment]
> @kennymckormick really appreciate it if you'd like to provide any update on supporting AVA-Kinetics. Thanks!

Hi, @coldmanck, currently I do not have any plans to support AVA-Kinetics.

**Comment by hukkai** [comment]
@Deep-learning999 @coldmanck Hi AVA-Kinetics is supported in the 1.x branch https://github.com/open-mmlab/mmaction2/tree/1.x/configs/detection/ava_kinetics
